config/src/test/java/io/confluent/common/config/ConfigUtilsTest.java

```diff
@@ -61,5 +63,20 @@ public void testTranslateDeprecated() throws Exception {
     assertEquals(new Integer(42), newProps.get("unexpected.non.string.object"));
     assertEquals(null, props.getProperty("unexpected.non.string.object"));
     assertEquals(new Integer(42), props.get("unexpected.non.string.object"));
+
+  }
+
+  @Test
+  public void testAllowsNewKey() throws Exception {
+    Properties props = new Properties();
+    props.setProperty("foo.bar", "baz");
+    Properties newProps = ConfigUtils.translateDeprecated(props, new String[][]{
+        {"foo.bar", "foo.bar.deprecated"},
+        {"chicken", "rooster", "hen"},
```

Comment:
not necessary, but actually having them in there tests that they don't affect the other props